homeworks/roman.vlasov_batteryfree/homework_3/function_array.js

```diff
@@ -0,0 +1,16 @@
+function degree(a) {
+    return a * a;
+}
+function arraystep(a, b) {
+    const arr = [];
+    for (let i = 0; i <= (parseInt(a / b, 10)); i++) {
```

Comment:
rethink and simplify you cycle:

1. try not to use parseInt (not needed)
2. try not to use division as condition (the is a simpler way)

hint: you could use i = i + step instead of i++